src/config/config.rs

```diff
@@ -682,6 +682,10 @@ pub struct Grpc {
     /// This refers to the gRPC method you're going to call. For instance
     /// `GetAllNews`.
     pub method: String,
+    #[serde(default, skip_serializing_if = "is_default")]
+    /// This refers to the gRPC message you're going to use for decoding
+    /// grpc-status-details-bin. For instance `google.rpc.Status`.
```

Comment:
I think `google.rpc.Status` is should be used anyway according to [doc](https://cloud.google.com/apis/design/errors#error_model) but the underlying error could be different. Or, actually, it could be set of errors since server can return different error, right?

Reply:
As I found, there is no real RFC that you should use only `google.rpc.Status` in the `grpc-status-details-bin`, technically, you can override the default behavior.
I've done this way because decided that if the user needs to pass this error header, he should explicitly add all the dependencies related to the Google imports.

But yes, you are right, I think most people will use `google.rpc.Status` and I'm not sure how often people customize this binary header.

The only thing I found, that is near to RFC is:
> Note that HTTP2 does not allow arbitrary octet sequences for header values so binary header values must be encoded using Base64 as per https://tools.ietf.org/html/rfc4648#section-4. Implementations MUST accept padded and un-padded values and should emit un-padded values. Applications define binary headers by having their names end with "-bin". Runtime libraries use this suffix to detect binary headers and properly apply base64 encoding & decoding as headers are sent and received.

Reply:
I think we can focus only on `google.rpc.Status` right now and try to parse details as Status:

- `google.rpc.Status` should be default value anyway for most cases and its recommended by google https://cloud.google.com/apis/design/errors#error_model
- if we get use case for implementing different type we can implement it later
- if parsing fails that not a critical problem and we still provide code and message in the response

we should use https://docs.rs/tonic/0.11.0/tonic/struct.Status.html and if required https://docs.rs/tonic-types/0.11.0/tonic_types/pb/struct.Status.html to parse error